src/pages/task/ActionButtons.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { useState } from "react";
 import { useMutation } from "@apollo/client";
 import styled from "@emotion/styled";
+import LeafyGreenButton from "@leafygreen-ui/button";
```

Comment:
I think we should be able to use the Button imported on line 8 since it's just a wrapper over `@leafygreen-ui/button` (maybe along with `@ts-expect-error`)

Reply:
our Button component doesn't accept a `to` or `as` prop. I created this ticket to simplify the component interface so it's easier to maintain. https://jira.mongodb.org/browse/EVG-15602. I made an attempt in c6b393f but it fails to apply the extra props, loading & dataCy.